main.go

```diff
@@ -54,31 +55,69 @@ func init() {
 }
 
 var flags = struct {
-	EnableLeaderElection bool
-	MetricsAddr          string
+	LeaderElect        bool
+	MetricsBindAddress string
 }{}
 
-func initFlags() {
-	flag.BoolVar(&flags.EnableLeaderElection, "leader-elect", false, "Enable leader election for controller manager.")
-	flag.StringVar(&flags.MetricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+func initFlags() (errors []error) {
```

Comment:
I decided to return slice of error here to print as many errors as possible so we don't have to redeploy multiple times just to discover the next error.